### PR TITLE
test: component description conformance test

### PR DESCRIPTION
**Pull request by levithomason**
This PR adds a failing conformance test for missing component descriptions.  It also attempts to add some heuristics around detecting an unhelpful component description.

### Workflow changes

Watch and rebuild component info JSON files during tests.  This enables changes to doc info, like descriptions, to be picked up and tested between tests.  Specifically, conformance tests can now be run in watch mode and successfully pick up on changes to documentation concerns.

### Tests added

1. has a docblock description
1. docblock description is long enough to be meaningful
2. docblock description is short enough to be quickly understood

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,11 +32,11 @@
...
     "start": "gulp --series dll docs",
     "satisfied": "satisfied --skip-invalid --ignore \"^typescript$\"",
-    "pretest": "yarn satisfied && gulp build:docs:docgen && gulp build:docs:component-menu-behaviors",
-    "test": "cross-env NODE_ENV=test jest --coverage",
+    "pretest": "yarn satisfied && gulp --series build:docs:docgen build:docs:component-menu-behaviors",
+    "test": "gulp test",
     "test:visual": "gulp screener",
     "test:projects:cra-ts": "gulp test:projects:cra-ts",
-    "test:watch": "yarn test --watchAll",
+    "test:watch": "gulp test:watch",
```
Comment (levithomason) [inline package.json:39]: This change was required to run jest's watch and gulp's doc watch commands in parallel.  The watchers never exit so they cannot be started in series.

**Review by alinais — APPROVED** [review]

**Review by miroslavstastny — COMMENTED** [review]
Review+ on the changes, but I am not approving as I made changes as well.

File: src/components/RadioGroup/RadioGroupItem.tsx [hunk 2]
```diff
@@ -45,6 +45,7 @@ export interface RadioGroupItemState {
 }
 
 /**
+ * A single radio with a radio group.
```
Comment (mnajdova) [inline src/components/RadioGroup/RadioGroupItem.tsx]: 'A single radio within a radio group.' might be better description

**Review by mnajdova — APPROVED** [review]
Approved with one suggestion for consideration in the RadioGroupItem's description.

**Pull request merged**